sdcm/utils/common.py

```diff
@@ -355,13 +392,61 @@ def inner(*args, **kwargs):
                 return_val = fun(*args, **kwargs)
                 LOGGER.debug("[{thread_name}] Done.".format(thread_name=thread_name))
                 return return_val
+            inner.__name__ = fun.__name__
```

Comment:
use `functools.warps()` instead of using __name__ like this, it would copy anything needed, including docstrings
https://docs.python.org/2/library/functools.html#functools.wraps

Reply:
thx, changed